### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by ancostas**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -2,17 +2,21 @@
 
 ## Local Development
 
-First of all, dowload [Unity3D](https://unity3d.com/get-unity/download) and [Visual Studio 2017](https://www.visualstudio.com/downloads/). 
+First of all, download [Unity3D](https://unity3d.com/get-unity/download), [Visual Studio 2017](https://www.visualstudio.com/downloads/), and [Vungle Unity Plugin](https://dashboard.vungle.com/sdk). 
 
-Next, checkout this repo and open the project in Unity3D. When you first open the project, Unity will import all the asset, so it will take a little while. Once done, you can select the start scene from Scenes -> Start Scene and click Play to start the game in Unity3D.
+Next, checkout this repo and open the project in __Unity3D__ (not Visual Studio 2017). When you first open the project, Unity will import all the asset, so it will take a little while. 
+
+After this, execute the downloaded `VunglePlugin-x.x.x.unitypackage` while Ruzik Odyssey is loaded in Unity. This will add the Vungle plugin to the project.
```
Comment (nsamteladze) [inline docs/getting-started.md]: You actually don't need to install plugins for the second time. You install them once and they become part of your solution and your source code. This is different from the usual Nuget package management experience everybody is used too. Definitely not ideal, but this is how unity plugins system works from what I understand. Bottom line is, you just pull the code and run in, no need to install plugins again.

Comment (nsamteladze) [reply docs/getting-started.md]: Let's also remove any reference of Vungle from here since I just removed it (see #21).

**Review by nsamteladze — CHANGES_REQUESTED** [review]
Definitely nice updates! Nobody ever tried to run this game except me before :) Let's just remove Vungle references.

File: docs/getting-started.md (hunk 1 above)
Comment (ancostas) [reply docs/getting-started.md:9]: @nsamteladze still tryna do this thing?

Comment (nsamteladze) [reply docs/getting-started.md:9]: @ancostas in a perfect world, yes. I still think this is a fun project to do. But, unfortunately, I won't have time most likely :( What about you?

Comment (ancostas) [reply docs/getting-started.md:9]: Last we spoke I think you wanted to make some kind of LLC so we can sell this thing, or?

**Pull request closed without merging**